server/src/building/UserProjectConfiguration.cpp

```diff
@@ -92,9 +92,10 @@ UserProjectConfiguration::RunProjectConfigurationCommands(const fs::path &buildD
             ShellExecTask::ExecutionParameters cmakeParams(
                 Paths::getCMake(),
                 cmakeOptionsWithMandatory);
+            // Use "--always-make" because bear catch only called commands
             ShellExecTask::ExecutionParameters bearMakeParams(
-                Paths::getBear(),
-                { Paths::getMake(), MakefileUtils::threadFlag() });
+                    Paths::getBear(),
+                    {Paths::getMake(), MakefileUtils::threadFlag(), "--always-make"});
```

Comment:
it is worth a separate PR and test process

Reply:
Move to separate PR